src/main/java/frc/robot/RobotContainer.java

```diff
@@ -45,15 +55,30 @@ private void configureButtonBindings() {
 
         swerve.setDefaultCommand(new TeleopSwerve(
             swerve,
-            () -> -m_driverController.getLeftX(),
-            () -> -m_driverController.getLeftY(),
-            () -> -m_driverController.getRightX(), 
-            () -> m_driverController.a().getAsBoolean()));
+            () -> driverController.getLeftX(),
+            () -> driverController.getLeftY(),
+            () -> driverController.getRightX(), 
+            () -> driverController.a().getAsBoolean())); //TODO: i'm not sure if this is right for toggling field/robot centric mode
```

Comment:
it's right, but it's not a toggle. it allows you to drive as robot relative only while holding down the button